scripts/deno/updateArticleViews.ts

```diff
@@ -0,0 +1,152 @@
+import * as path from "https://deno.land/std@0.102.0/path/mod.ts";
+import { load } from "https://deno.land/std@0.212.0/dotenv/mod.ts";
+import { createClient } from "npm:next-sanity";
+import { clientConfig } from "../../aksel.nav.no/website/sanity/config.ts";
+import { amplitudeFetchJSON, hashString, queryArticleURLs } from "./utils.ts";
+
+// this part is a bit silly: script "plumbing" essentially. ModuleDir is the root of this
+// current file, it's used so relative paths from this file into the rest of the repo will
+// work as expected regardless of where it is called from (disregarding CWD of caller)
+// it's a WEB-first API, so we get files in URL format, these must be converted to a path
+const mainModuleDir = path.dirname(path.fromFileUrl(Deno.mainModule));
+await load({
+  envPath: `${mainModuleDir}/../../aksel.nav.no/website/.env`,
+  export: true,
+});
+
+type ViewData = {
+  views_day: number;
+  views_week: number;
+  views_month: number;
+  views_year: number;
+};
+
+const view_datas = new Map<string, ViewData>();
+
+const _queries = [
+  await amplitudeFetchJSON("e-f5vcqiqk"), // last 24h
+  await amplitudeFetchJSON("e-69mzf301"), // last 7d
+  await amplitudeFetchJSON("e-wwh3n9l1"), // last 30d
+  await amplitudeFetchJSON("e-rzw8jk6j"), // last 365d
+];
+// const queries = await Promise.all(_queries); // can't do this... it overloads the Proxy
+const queries = _queries;
+
+const json_obj_24h = queries[0];
+const json_obj_7d = queries[1];
+const json_obj_30d = queries[2];
+const json_obj_365d = queries[3];
+for (const [idx, view_entry] of json_obj_24h.data.series.entries()) {
```

Comment:
Kan logikken til disse `for`-loopene trekkes ut til en utility slik at man ikke trenger å f.eks repetere logikk for `.reduce`?